lib/bunch/typespec.ex

```diff
@@ -26,7 +26,8 @@ defmodule Bunch.Typespec do
       :b
 
   """
-  defmacro @{:list_type, _, [{:"::", _, [{name, _, _} = name_var, list]}]} do
+  # credo:disable-for-next-line Credo.Check.Consistency.UnusedVariableNames
+  defmacro @{:list_type, _, [{:"::", _, [{name, _, _env} = name_var, list]}]} do
```

Comment:
Not sure if better, but that's what I usually do

```suggestion
  defmacro @{:list_type, _meta1, [{:"::", _meta2, [{name, _meta3, _env} = name_var, list]}]} do
```